src/features/stock-out.js

```diff
@@ -263,7 +263,15 @@ async function getStockOutConfigs({
       type: 'input',
       name: 'form_name',
       message: 'Enter stock out form ID',
-      default: 'stock_out'
+      default: 'stock_out',
+      when: function (answers){
+        const argv = process.argv;
+        if (!argv[5]){
+          return true;
+        } 
+        answers.form_name = argv[5];
```

Comment:
Well done.

We could sanitize user inputs by escaping potentially harmful characters. Using for example escape from [validator](https://www.npmjs.com/package/validator).

`answers.form_name = validator.escape(argv[5]);`

This comment is valid for every input from user.

Reply:
For sure.